specification/security/resource-manager/Microsoft.Security/preview/2022-01-01-preview/ServerVulnerabilityAssessmentsSettings.json

```diff
@@ -0,0 +1,328 @@
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete the server vulnerability assessments setting of the kind settingKind from the subscription": {
+            "$ref": "./examples/ServerVulnerabilityAssessmentsSettings/DeleteServerVulnerabilityAssessmentsSetting_example.json"
+          }
+        },
+        "tags": [
+          "ServerVulnerabilityAssessmentsSettings"
+        ],
+        "description": "Delete the server vulnerability assessments setting of the requested kind from the subscription",
+        "operationId": "ServerVulnerabilityAssessmentsSettings_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SettingKind"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Setting was deleted"
+          },
+          "204": {
+            "description": "No Content - Setting does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerVulnerabilityAssessmentsSettingsList": {
+      "type": "object",
+      "description": "A page of a server vulnerability assessments settings list",
+      "properties": {
+        "value": {
+          "description": "A collection of server vulnerability assessments settings in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServerVulnerabilityAssessmentsSetting"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "ServerVulnerabilityAssessmentsSetting": {
+      "type": "object",
+      "description": "A base vulnerability assessments setting on servers in the defined scope.",
+      "properties": {
+        "kind": {
+          "description": "The kind of the server vulnerability assessments setting.",
+          "$ref": "#/definitions/ServerVulnerabilityAssessmentsSettingKind"
+        },
+        "systemData": {
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
```

Comment:
>Resource [](http://example.com/codeflow?start=62&length=8)

Would suggest inheriting ProxyResource instead because its clearer in intent and you'll probably get less linter warnings or it.